prometheus-collectors/class-login-stats-collector.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Automattic\VIP\Prometheus;
+
+use Prometheus\Counter;
+use Prometheus\RegistryInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class Login_Stats_Collector implements CollectorInterface {
+	private Counter $login_limit_exceeded_counter;
+	private Counter $password_reset_limit_exceeded_counter;
+	private Counter $successful_login_counter;
```

Comment:
Just wondering, but why do we need a counter for successful logins? The other counters are more for failed/rate-limited requests.

Reply:
I was thinking on using the rate() function (or maybe something similar) in Prometheus to see the spikes in the failed to successful login ratio.

The counter is cheap, we can always remove it if we find that we don't need it.

Reply:
OK. What would the ratio of failed to successful would be helpful in determining?

Reply:
For one that would show us abnormal patterns?